DNN Platform/DotNetNuke.Web.Mvc/Helpers/DnnUrlHelper.cs

```diff
@@ -77,7 +93,7 @@ public virtual bool IsLocalUrl(string url)
 
         public virtual string Action()
         {
-            return _viewContext.RequestContext.HttpContext.Request.RawUrl;
+            return UrlHelper.RequestContext.HttpContext.Request.RawUrl;
```

Comment:
I updated this code to normalize the 2 code paths we have in the `DnnUrlHelper`. Now regardless if you are using the `viewContext` or `requestContext` this method generates the url using the same `RequestContext` path which is retrieved from the `UrlHelper`